support/xspec-dev/mvn-xproc-xspec-html.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# shellcheck source=../common/subcommand_common.bash
+
+source "$SCRIPT_DIR/../../src/common/subcommand_common.bash"
+
+XPROC_FILE="${SCRIPT_DIR}/xspec-single-report.xpl"
+
+usage() {
+    cat <<EOF
+Usage: ${BASE_COMMAND:-$(basename "${BASH_SOURCE[0]}")} XSPEC_FILE [ADDITIONAL_ARGS]
+
+Applies XSpec Xproc to XSpec file and produces an HTML report of the test results.
+
+Additional arguments for XML Calabash should be specified in the 'key=value' format.
+
+EOF
+}
+
+[[ -z "${1-}" ]] && { echo "Error: XSPEC_FILE not specified"; usage; exit 1; }
+XSPEC_FILE=$1
+
+ADDITIONAL_ARGS=$(shift 1; echo "${*// /\\ }")
+
+RESULT_FILE="xspec/$( echo $(basename "${XSPEC_FILE%.*}") ).html"
```

Comment:
Is there a reason that the output goes into a path relative to the shell script that is with a hard-coded prefix? If you use it outside the presumed directory structure and if the xspec folder fails it will fail if you do not create it (the current shell script does not). I would just put it in the adjacent directory and switch out the extension.

```suggestion
RESULT_FILE="${XSPEC_FILE%%.*}.html"
```

As a dev user, I also presume I am not unique in that I would run the command and expect to see the file, _as a default_ adjacent to the current folder in which I am operating from (so wherever the target XSpec file is). That did not happen when I was re-evaluating the updated branch for this PR, and had to run the shell script again quickly in debug mode to understand where the file was. This violates POLA/POLS, or the [principle of least surprise](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).

Reply:
Now `{basename}-result.html` cf https://github.com/usnistgov/metaschema-xslt/pull/94/commits/4b2b86197f2b62a12e55b036607bf0e5378f694f